### ISSUE TITLE
NotAbleToComment

### ISSUE THREAD

**Issue by tavisca-agupta** [k0shk0sh/FastHub#2578]
**FastHub Version: 4.6.7**  
**Android Version: 9 (SDK: 28)**  
**Device Information:**  
- **Manufacturer:** OPPO  
- **Brand:** OPPO  
- **Model:** realme 2 Pro  
---

 Iam not able to reply back ,to the commenter in the conversation window (of pull request).
It shows "not found" as error.
Thanks and regards
Aman

**Comment by yakov116** [comment]
@tavisca-agupta we will need more info

**Comment by tavisca-agupta** [comment]
So,i will try to address this issue by sending some images as a reference.
Iam not able to comment from particular section of the app.

![](https://i.imgur.com/zhQ8dm4.jpg)

Above one is entry point.

![](https://i.imgur.com/or4u4vZ.jpg)

And this is error shown by the app.

Thanks and regards.

**Comment by yakov116** [comment]
@tavisca-agupta your on the latest app correct?

**Comment by tavisca-agupta** [comment]
@yakov116 yes,same problem persist in latest version of the app
![](https://i.imgur.com/pteSHg8.jpg)

With error(below):
![](https://i.imgur.com/TscZ3Dg.jpg)

**Comment by yakov116** [comment]
@tavisca-agupta is it an organization repository?

**Comment by tavisca-agupta** [comment]
Yes its a organization's public repository,and everything is working fine ,even comments also,but only above mentioned reply part is not working properly.

**Comment by TysonMN** [comment]
I think this issue is a duplicate of issue #1820 but also contains more information.

**Comment by TysonMN** [comment]
I fixed this problem by logging into FastHub using a personal authentication token.
